delete_module.php

```diff
@@ -24,12 +24,11 @@
  */
 
 require_once(__DIR__ . '/../../../config.php');
-require_once(__DIR__.'/lib.php');
-require_once($CFG->libdir.'/gradelib.php');
-require_once($CFG->libdir.'/completionlib.php');
-require_once($CFG->dirroot.'/blog/lib.php');
+require_once(__DIR__.'/classes/reporter.php');
 require_login();
 
+use tool_fix_delete_modules\reporter as reporter;
```

Comment:
Here and in other places can this be just `use tool_fix_delete_modules\reporter;`?